### ISSUE TITLE
The provided Link is not working

### ISSUE THREAD

**Issue by irtazas** [lilhuang/FindingFeynman#1]
The "Try it here!" link isn't working.
I tried to download the zip file to run it locally but couldn't figure out how to run it aswell.

**Comment by TomHodson** [comment]
Hey! Sorry, the link probably won't go up anytime soon. You should be able to play it by first compiling it with grunt then serving that directory with a local file server like "python3 -m http.server" 

Sadly it was a game we made in a 2 day hackathon and we didn't get very far on it, so even if you get it to run, it doesn't really do anything.  Was there something specific you were interested in?

**Comment by irtazas** [comment]
I was looking into ways to make Feynman diagrams more accessible to students. I thought this game could be a good starting point. However as it is not really functional at this stage I don't think I am going to be able to use it. Thanks anyway.

**Comment by TomHodson** [comment]
Yeah it's a shame we never finished it. I hope you find an alternative.
